### PR TITLE
{cae}[foss/2016b] CGNS 3.3.1

### PR DESCRIPTION
**Pull request by iotaka**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/c/CGNS/CGNS-3.3.1-foss-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+description = """The CGNS system is designed to facilitate the exchange of data between sites and applications, and to help stabilize the archiving of aerodynamic data."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['https://github.com/CGNS/CGNS/archive/v%(version)s.tar.gz']
+sources = [SOURCE_TAR_GZ]
+checksums = ['65c55998270c3e125e28ec5c3742e15d']
+
+easyblock = 'CMakeMake'
```
Comment (pescobar) [inline easybuild/easyconfigs/c/CGNS/CGNS-3.3.1-foss-2016b.eb]: I guess you should move this line to beginning of the easyconfig

File: easybuild/easyconfigs/c/CGNS/CGNS-3.3.1-foss-2016b.eb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+name = 'CGNS'
+version = '3.3.1'
+
+homepage = 'https://cgns.github.io/'
+description = """The CGNS system is designed to facilitate the exchange of data between sites and applications, and to help stabilize the archiving of aerodynamic data."""
```
Comment (pescobar) [inline easybuild/easyconfigs/c/CGNS/CGNS-3.3.1-foss-2016b.eb]: this line should be shorter. Split it in two lines

**Comment by pescobar** [comment]
this easyconfig is missing the `sanity_check_paths`

**Comment by iotaka** [comment]
@pescobar no idea for the `sanity_check_paths`

**Comment by pescobar** [comment]
@iotaka check the contents of the installation directory after a successful build and add something similar to this

https://github.com/easybuilders/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/b/BLAST/BLAST-2.2.26-Linux_x86_64.eb#L22

**Comment by pescobar** [comment]
Test report by @pescobar
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
login20.cluster.bc2.ch - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/a08ef356ac5a549030ffbae6175d46fb for a full test report.

**Comment by pescobar** [comment]
@iotaka add this sanity check

```
sanity_check_paths = {
    'files': ["bin/%s" % x for x in ["cgnscheck", "cgnscompress", "cgnsconvert", "cgnsdiff",  
                                     "cgnslist, "cgnsnames", "cgnsupdate"]],
    'dirs': [],
}
```

**Comment by pescobar** [comment]
@iotaka I did a mistake, excuse me. Please use this sanity check

```
sanity_check_paths = {
    'files': ["bin/%s" % x for x in ["cgnscheck", "cgnscompress", "cgnsconvert", "cgnsdiff",
                                     "cgnslist", "cgnsnames", "cgnsupdate"]],
    'dirs': [],
}


```

**Comment by pescobar** [comment]
Test report by @pescobar
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
login20.cluster.bc2.ch - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/36f3c2d9e2616880fe72b04f17c11fc3 for a full test report.

**Comment by pescobar** [comment]
this one seems ready to merge. 

The only style change I would do is to move `moduleclass = 'cae'` to the bottom of the easyconfig

**Comment by pescobar** [comment]
Test report by @pescobar
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
login20.cluster.bc2.ch - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/7d38d310c7a27d36c6434db6e1fc20fd for a full test report.

**Comment by iotaka** [comment]
thank you @pescobar  :)

File: easybuild/easyconfigs/c/CGNS/CGNS-3.3.1-foss-2016b.eb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+homepage = 'https://cgns.github.io/'
+description = """The CGNS system is designed to facilitate the exchange of data between sites and applications,
+ and to help stabilize the archiving of aerodynamic data."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['https://github.com/CGNS/CGNS/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['65c55998270c3e125e28ec5c3742e15d']
```
Comment (boegel) [inline easybuild/easyconfigs/c/CGNS/CGNS-3.3.1-foss-2016b.eb]: @iotaka Do you mind making this a SHA256 checksum?

We're trying to move towards making that a standard, and been requiring SHA256 checksums for recent pull requests.

Should be `81093693b2e21a99c5640b82b267a495625b663d7b8125d5f1e9e7aaa1f8d469`

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2021.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/396d1ae4ee19f62fe831d6885979a77b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2474.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/88c29f275422a67eb8c6ef63d2209047 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @iotaka!

**Pull request merged**